src/js/utils/StringUtil.js

```diff
@@ -215,6 +215,10 @@ const StringUtil = {
     }, []);
 
     return this.idToTitle(splitID, splitBy, replace, removeConsecutive);
+  },
+
+  getLongDashPlaceholder() {
```

Comment:
This isn’t really what I had in mind and I’m sorry that I wasn’t precise. Having a util like this one is still not better then a constants as it comes with an overhead ans is be no means more declarative than using a constants. If you want to go with an util I’d opt for a more general util similar to an either functor that either returns the _value_ or the placeholder.

Reply:
addressed, I went back with hard coded as the other PRs have.